include/PPN-microbench/constants.hpp

```diff
@@ -13,4 +16,6 @@ typedef uint8_t u8;
 typedef int64_t i64;
 typedef int32_t i32;
 typedef int16_t i16;
-typedef int8_t i8;
\ No newline at end of file
+typedef int8_t i8;
```

Comment:
Prefer `using` declarations.
Also, consider using short hands for all other primitive data types. 
```suggestion
using i64 = int64_t;
using i32 = int32_t;
using i16 = int16_t;
using i8  = int8_t;
using isz = ssize_t;

using u64 = uint64_t;
using u32 = uint32_t;
using u16 = uint16_t;
using u8  = uint8_t;
using usz = size_t;

using f32 = float;
using f64 = double;
```